cif_core.dic

```diff
@@ -18456,6 +18480,11 @@ save_publ_author.id
     _type.source                  Related
     _type.container               Single
     _type.contents                Word
+    _method.purpose               Definition
+    _method.expression
+;
+    _enumeration.default = Current_Row(publ_author) + 1
```

Comment:
This note serves as a reminder to check if the naming and usage of `Current_Row()` matches the one agreed upon in https://github.com/COMCIFS/dREL/issues/6.